src/components/core/base/VisualizationLoadingHOC.tsx

```diff
@@ -281,15 +288,42 @@ export function visualizationLoadingHOC<
             }
         };
 
+        private getAllMeasureHeaderItems(
```

Comment:
Isn't this common-looking selector already present in some service?

Reply:
did not find it enywhere